Крестики-нолики_1/PlayField.cpp

```diff
@@ -0,0 +1,96 @@
+#include "PlayField.h"
+#include <iostream>
+#include <vector>
+#include <cassert>
+
+using namespace std;
+
+vector < PlayField::CellPos > PlayField::getEmptyCells() const
+{
+	vector<CellPos> emptyCell;
+	for (int i = 0; i < fieldSize * fieldSize; i++)
+	{
+		if (cellsState[i] == csEmpty)
+		{
+			CellPos cell = CellPos::GetCellPos(i);
+			emptyCell.push_back(cell);
+		}
+	}
+	return emptyCell;
+}
+
+PlayField PlayField::makeMove(CellPos cellPos)
+{
+	if (newCell == PlayField::csNought)
+		newCell = PlayField::csCross;
+	else newCell = PlayField::csNought;

```

Comment:
```suggestion
	if (newCell == PlayField::csNought)
		newCell = PlayField::csCross;
	else 
	        newCell = PlayField::csNought;

```
Так выглядит лучше. Но надо избавится от newCell, потому что это величина производная от состояния поля.